ddtrace/contrib/aioredis/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""
+The aioredis integration instruments aioredis requests. Version 2.0 and above are fully
+supported. In version 1.3, basic commands are supported but pipeline
```

Comment:
We have to update this docstring now that 1.3 is supported fully now too.

Reply:
Thanks for the suggestions 😄

Reply:
this wasn't addressed